test/fungus_toast/games/engine_test.exs

```diff
@@ -0,0 +1,41 @@
+defmodule FungusToast.Games.EngineTest do
+  alias FungusToast.Games.Engine
+  use ExUnit.Case, async: true
+  doctest Engine
+
+  describe "single player game creation" do
```

Comment:
Let's add "userName" to the POST /api/games endpoint, so we have the username of the player who initiated the game. I think the player object that comes back will need a username (to display) for each player as well.

(PS: this probably isn't the ideal line of code to add this comment, but it was the first place where I saw something identifiable from the API :P)